en/docs/references/preview-features.md

```diff
@@ -0,0 +1,13 @@
+# Preview Features
+
+Preview features offer optional functionality that you can enable if required.
+
+To enable Choreo features that are released as preview features, follow these steps:
+
+1. Expand the profile menu and click **Feature Preview**.
+
+   ![Feature preview](../assets/img/feature-preview/feature-preview.png){.cInlineImage-full}
```

Comment:
The images seem to be missing when I try it locally

![image](https://user-images.githubusercontent.com/8376040/185425969-c3968d18-6160-440f-a9ce-9f82586f488a.png)

Reply:
Committed the missing images.